userspace/libsinsp/container_engine/docker/docker_linux.cpp

```diff
@@ -30,7 +30,7 @@ constexpr const cgroup_layout DOCKER_CGROUP_LAYOUT[] = {{"/", ""},  // non-syste
                                                         {nullptr, nullptr}};
 }
 
-std::string docker_linux::m_docker_sock = "/var/run/docker.sock";
+std::string docker_linux::s_docker_sock = "/var/run/docker.sock";
```

Comment:
Since this is a static, rename it like we use to call static members, with `s_` prefix.